cmd/cleanup.go

```diff
@@ -46,6 +60,20 @@ func cleanup(openSubdomain string) {
 	if err != nil {
 		reportError(err.Error(), true)
 	}
-	fmt.Println("Successfully closed tunnel")
+	fmt.Printf("%s%s\n", "Successfully closed tunnel for subdomain: ", openSubdomain)
+}
+
+func cleanupAll() {
+	response, err := restAPI.ListSubdomainAPI()
```

Comment:
why not use the list tunnel api?

Reply:
I wanted to reuse the `cleanup` function, using `ListSubdomainAPI()` allowed me to get the subdomain name and pass it to it easily. 

You're right though, I could've also used the list tunnel api